docs/specification/README.md

```diff
@@ -33,9 +33,10 @@ Some helpful terms to understand in the following sections are:
 - **Message**: A type of record representing a timestamped message on a channel (and therefore associated with a topic/schema). A message can be parsed by a reader that has also read the channel info for the channel on which the message appears.
 - **Chunk**: A record type that wraps a compressed set of channel info and message records.
 - **Attachment**: Extra data that may be included in the file, outside the chunks. Attachments may be quickly listed and accessed via an index at the end of the file.
-- **Index**: The format contains indexes for both messages and attachments. For messages, there are two levels of indexing - a **Chunk Index** at the end of the file points to chunks by offset, enabling fast location of chunks based on topic and timerange. A second index - the **Message Index** - after each chunk contains, for each channel in the chunk, and offset and timestamp for every message to allow fast location of messages within the uncompressed chunk data.
-
-  The attachment index at the end of the file allows for fast listing and location of attachments based on name, timestamp, or attachment type.
+- **Index**: The format contains indexes for both messages and attachments. For messages, there are two levels of indexing - a **Chunk Index** at the end of the file points to chunks by offset, enabling fast location of chunks based on topic and timerange. A second index - the **Message Index** - after each chunk contains, for each channel in the chunk, and offset and timestamp for every message to allow fast location of messages within the uncompressed chunk data. The attachment index at the end of the file allows for fast listing and location of attachments based on name, timestamp, or attachment type.
+- **Statistics**: A type of record at the end of the file, used to support fast summarization of file contents.
```

Comment:
Why does statistics record get a call-out here but other records don't? Actually looking at all these terms it seems like there is some overlap between defining records and non-record concepts.

Reply:
I called it out because I had to reference it elsewhere in the section